### PR TITLE
UX Icon: updated fas-question icon

### PR DESCRIPTION
**Pull request by NidhiKJha**
## Explanation

Replaced instances of the `fas-question icon`  with `ICON.QUESTION`

## Pre-merge author checklist

- [ ] I've clearly explained:
  - [ ] What problem this PR is solving
  - [ ] How this problem was solved
  - [ ] How reviewers can test my changes
- [ ] Sufficient automated test coverage has been added

## Pre-merge reviewer checklist

- [ ] Manual testing (e.g. pull and build branch, run in the browser, test code being changed)
- [ ] PR is linked to the appropriate GitHub issue
- [ ] **IF** this PR fixes a bug in the release milestone, add this PR to the release milestone

If further QA is required (e.g. new feature, complex testing steps, large refactor), add the `Extension QA Board` label.

In this case, a QA Engineer approval will be be required.

### CODE REVIEW

File: ui/components/ui/form-field/form-field.stories.js [hunk 1]
```diff
@@ -94,6 +95,7 @@ export const CustomComponents = (args) => {
     <div style={{ width: '600px' }}>
       <FormField
         {...args}
+        titleHeadingWrapperProps={{ alignItems: AlignItems.center }}
```
Comment (NidhiKJha) [inline ui/components/ui/form-field/form-field.stories.js:98]: Adding `titleHeadingWrapperProps` as the fallback prop to add props to `Box` here.

### After

afhb

### Before
hhgv

File: ui/components/ui/form-field/form-field.js [hunk 2]
```diff
@@ -210,6 +212,13 @@ FormField.propTypes = {
   titleDetailWrapperProps: PropTypes.shape({
     ...Box.propTypes,
   }),
+  /**
+   * Props to pass to wrapping Box component of the titleHeading component
+   * Accepts all props of the Box component
+   */
+  titleHeadingWrapperProps: PropTypes.shape({
+    ...Box.propTypes,
+  }),
```
Comment (NidhiKJha) [inline ui/components/ui/form-field/form-field.js:221]: Adding `titleHeadingWrapperProps` as the fallback prop to add props to `Box` here. cc @georgewrmarshall

File: ui/components/app/custom-spending-cap/custom-spending-cap-tooltip.js [hunk 3]
```diff
@@ -31,7 +32,7 @@ export const CustomSpendingCapTooltip = ({
       {tooltipIcon ? (
         <i className="fa fa-exclamation-triangle form-field__heading-title__tooltip__warning-icon" />
       ) : (
-        tooltipIcon !== '' && <i className="fa fa-question-circle" />
+        tooltipIcon !== '' && <Icon name={ICON_NAMES.QUESTION} />
       )}
     </Tooltip>
   </Box>
```
Comment (georgewrmarshall) [inline ui/components/app/custom-spending-cap/custom-spending-cap-tooltip.js:38]: Screenshot 2023-02-21 at 10 04 13 PM

File: ui/components/ui/new-network-info/new-network-info.js [hunk 4]
```diff
@@ -103,7 +104,11 @@ const NewNetworkInfo = () => {
           primaryTokenImage ? (
             <Identicon image={primaryTokenImage} diameter={14} />
           ) : (
-            <i className="fa fa-question-circle" />
+            <Icon
+              className="question"
+              name={ICON_NAMES.QUESTION}
+              color={Color.iconDefault}
+            />
           )
```
Comment (georgewrmarshall) [inline ui/components/ui/new-network-info/new-network-info.js:112]: Could use screenshot here

Comment (georgewrmarshall) [inline ui/components/ui/new-network-info/new-network-info.js:112]: Could use screenshot here

File: ui/components/ui/review-spending-cap/review-spending-cap.js [hunk 5]
```diff
@@ -92,7 +93,11 @@ export default function ReviewSpendingCap({
...
               {Number(tokenValue) === 0 && (
-                <i className="far fa-question-circle review-spending-cap__heading-title__tooltip__question-icon" />
+                <Icon
+                  className="review-spending-cap__heading-title__tooltip__question-icon"
+                  name={ICON_NAMES.QUESTION}
+                  color={Color.iconDefault}
+                />
               )}
             </Tooltip>
           </Box>
```
Comment (georgewrmarshall) [inline ui/components/ui/review-spending-cap/review-spending-cap.js:103]: Adding screenshot for other reviewers

Screenshot 2023-02-21 at 10 08 28 PM

**Review by georgewrmarshall — COMMENTED** [review]
Looking good! Could use one more screenshot of the `ui/components/ui/new-network-info/new-network-info.js` you may have to find where it's used in the app for the screenshot or create a story and add it to this PR. I've add some screenshots in the comments for other reviewers

File: ui/components/ui/new-network-info/new-network-info.js (hunk 4 above)
Comment (NidhiKJha) [reply ui/components/ui/new-network-info/new-network-info.js:112]: Screenshot 2023-02-22 at 5 01 59 PM

**Review by NidhiKJha — COMMENTED** [review]
Screenshot for New Network Info

**Review by georgewrmarshall — APPROVED** [review]
LGTM!

**Review by darkwing — APPROVED** [review]

**Pull request merged**